pom.xml

```diff
@@ -198,6 +198,20 @@
         <module>testing/trino-tests</module>
     </modules>
 
+    <repositories>
+        <repository>
+            <id>apache.snapshots</id>
+            <name>Apache Development Snapshot Repository</name>
+            <url>https://repository.apache.org/content/repositories/snapshots/</url>
```

Comment:
of course temporary, not going to be merged

Reply:
correct, this is only as long as we're testing with the latest snapshot version of Iceberg